modules/my_health/docs/openapi.yaml

```diff
@@ -92,6 +92,43 @@ paths:
       summary: /v1/messaging/folders/{id}/messages
       parameters:
         - $ref: '#/components/parameters/InflectionHeader'
+  /v1/messaging/folders/{id}/search:
+    post:
+      description: |
```

Comment:
It would be good to clarify (either here or in the SecureMessageSearchRequest.yml schema) what the behavior is if multiple search params are specified. I assume they are AND-ed together?

Reply:
Also - are search results paginated in any way?

Reply:
I see page/per_page parameters in the controller but not described here.

Reply:
I updated the description to indicate that results must match ALL params. Also specified page and per_page as parameters to the request